test/extended/util/framework.go

```diff
@@ -1949,6 +1949,17 @@ func GetRouterPodTemplate(oc *CLI) (*corev1.PodTemplateSpec, string, error) {
 }
 
 func FindRouterImage(oc *CLI) (string, error) {
+	microShift, err := IsMicroShiftDeployment(oc)
```

Comment:
Leftover? `IsMicroShiftDeployment` is never defined.

Reply:
Done